apps/src/javalab/JavalabEditor.jsx

```diff
@@ -360,6 +362,40 @@ class JavalabEditor extends React.Component {
     });
   }
 
+  // This moves the active tab to the left in the tab menu
+  moveTabLeft() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
+    let index = newTabs.indexOf(activeTabKey);
+    if (index > 0) {
+      this.swapTabsSetKeyCloseTab(index - 1, index, newTabs);
+    }
+  }
+
+  // This moves the active tab to the right in the tab menu
+  moveTabRight() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
+    let index = newTabs.indexOf(activeTabKey);
+    if (index < newTabs.length - 1) {
+      this.swapTabsSetKeyCloseTab(index, index + 1, newTabs);
```

Comment:
Does this work if the adjacent file is support/validation?

Reply:
Same question!

Reply:
Yes, it does :)
In levelbuilder, when both support and validation files are visible, the reordering of tabs works. Screencast is below from follow-up PR: 
https://user-images.githubusercontent.com/107423305/191570879-53e36671-5beb-406b-a756-44717cd81a13.mp4